wiki/Ranking_Criteria/osu!mania/de.md

```diff
@@ -1,91 +1,205 @@
+- **Vermeide das Spammen von kurzen Kiai-Zeitabschnitten.** Dies kann ablenkend wirken und den Spielablauf negativ beeinflussen.
+- **Pausen sollten nur dann eingelegt werden, wenn ein Abschnitt der Musik extrem ruhig oder leise ist.** Bei niedrigeren Schwierigkeitsgraden, insbesondere bei Liedern mit höherer BPM, können Pausen verwendet werden, um Teile des Liedes zu vermeiden, die für neue Spieler unangemessen sind.
+
+## Schwierigkeitsgradabhängig
+
+Schwierigkeitsgradabhängige Regeln und Richtlinien treffen nur auf die Schwierigkeitsgrade zu, die aufgelistet sind und *gelten somit nicht für **jeden** osu!mania Schwierigkeitsgrad*. Rhythmus bezogene Regeln und Richtlinien gelten für Beatmaps mit etwa 180 BPM. Sollte dein Song drastisch schneller oder langsamer sein, so können einige Variablen anders sein, wie unter [Skalierung der BPM in den Ranking Kriterien](/wiki/Ranking_Criteria/Scaling_BPM) beschrieben wird.
+
+*Key-Mode-spezifische Richtlinien basieren auf den am häufigsten verwendeten Tastenmodi (4 Key und 7 Key). Alle anderen Key Modes sollten diese Richtlinien befolgen, wenn sie anwendbar sind.*
+
+### ![](/wiki/shared/diff/easy-m.png?20211215) Easy
+
+#### Regeln
+
+- **Slider-Velocity-Änderungen, die die Scroll-Geschwindigkeit der Beatmap verändern, sind nicht erlaubt.** Slider-Velocity-Änderungen zur Vereinheitlichung der Scroll-Geschwindigkeit in Beatmaps mit unterschiedlicher BPM sind ausgenommen.
+- **Bei Beatmaps mit unterschiedlicher BPM muss eine Scroll-Normalisierung vorgenommen werden.**
+
+#### Richtlinien
+
+- **HP und OD Werte sollten nicht höher als 7 sein.**
+- **Taktteilung von 1/4 und höher sollten nicht aufeinanderfolgend verwendet werden.** Langsamere Rhythmen sind für diesen Schwierigkeitsgrad besser geeignet.
+- **Lange Noten sollten für mindestens einen Beat gehalten werden.** Schnellere Release-Zeiten sind für Spieler dieses Niveaus unangemessen.
+- **Release-Zeiten von langen Noten sollten mindestens einen Beat von Anderen entfernt sein.** Kleinere Abstände zwischen den Auslösungen erfordern ein Maß an Fingerunabhängigkeit, welche Spieler dieses Niveaus nicht bewältigen können.
+- **Objekte sollten nicht während der Haltedauer einer 1/1 langen Note platziert werden.** Jedoch ist die Verwendung eines Objekts am Anfang oder Ende einer langen Note in Ordnung.
+- **Die Notendichte sollte hauptsächlich aus 1/1, gelegentlich 1/2, oder langsameren Rhythmen bestehen.**
+- **Vermeide die Verwendung von mehr als 5 aufeinanderfolgenden 1/2-Noten.**
+- **Vermeide die Verwendung von Chords bei aufeinanderfolgenden 1/2-Noten-Patterns.** Jedoch ist die Verwendung eines Chords, um ein Pattern zu beginnen oder zu beenden, in Ordnung.
```

Comment:
Opening this for discussion: should the wiki use `Akkorde` instead of `Chords`? I think the meaning should still be clear or am I overlooking something?

Reply:
I don't know. I've never seen anyone say `Akkorde` in the German osu!mania discord server.

Reply:
Yeah, I thought so too. Fine to leave as-is then. I always try to find fitting translations for words but this sounds rather forced, which should be avoided.